### PR TITLE
Feature/esm build lib

### PR DESCRIPTION
**Pull request by LucaMele**
The goal is to have one mjs that contains everything in the npm build

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (AndyOGo) [inline .babelrc]: jipiiii

File: build/webpack.config.js [hunk 2]
```diff
@@ -68,41 +68,6 @@ config.module.rules.push({
     loader: 'babel-loader',
     query: {
       cacheDirectory: false,
-      plugins: [
```
Comment (AndyOGo) [inline build/webpack.config.js]: 👍 💪

File: package.json [hunk 3]
```diff
@@ -2,12 +2,15 @@
   "name": "@axa-ch/aletheia",
   "version": "0.0.2-beta.11",
   "description": "AXA UI Renderer",
+  "module": "lib/main.es.js",
```
Comment (AndyOGo) [inline package.json]: 👍

File: package.json [hunk 4]
```diff
@@ -68,9 +71,14 @@
   },
   "devDependencies": {
     "async-waterfall": "^0.1.5",
-    "babel-core": "^6.26.0",
+    "autoprefixer": "^8.4.1",
```
Comment (AndyOGo) [inline package.json]: Nice

File: src/registries/IconRegistry.js [hunk 5]
```diff
@@ -14,8 +15,8 @@ import Check from '../assets/icons/check.svg'
 import Car from '../assets/icons/car.svg'
 import GoodsTransport from '../assets/icons/goods-transport.svg'
 import OtherVehicles from '../assets/icons/other-vehicle.svg'
-import Add from '@axa-ch/patterns-library/src/assets/icons/add.svg'
-import More from '@axa-ch/patterns-library/src/assets/icons/more.svg'
+import Add from '../../node_modules/@axa-ch/patterns-library/src/assets/icons/add.svg'
```
Comment (AndyOGo) [inline src/registries/IconRegistry.js]: please import from `@axa-ch/patterns-library`

File: src/styles/app.scss [hunk 6]
```diff
@@ -1,6 +1,6 @@
-@import "~@axa-ch/patterns-library/src/styles/reboot";
-@import "~@axa-ch/patterns-library/src/styles/fonts";
-@import "~@axa-ch/patterns-library/src/styles/trumps/visibility";
+@import "../../node_modules/@axa-ch/patterns-library/src/styles/reboot";
```
Comment (AndyOGo) [inline src/styles/app.scss]: should use node-sass [`includePaths`](https://github.com/sass/node-sass#includepaths)

**Comment by mjenny** [comment]
Fixes #41

**Pull request merged**